### PR TITLE
Refactor parts of lithosphere_mask.cc

### PR DESCRIPTION
**Pull request by SophieCoulson**
I have rearranged lithosphere_mask.cc so functions associated with finding the lab depth can be called from another plug in.

### CODE REVIEW

File: include/aspect/initial_temperature/lithosphere_mask.h [hunk 1]
```diff
@@ -33,6 +33,84 @@ namespace aspect
...
+          /**
+           * Initialization function. This function is called once at the
+           * beginning of the program. Checks preconditions.
+           */
+          void
+          initialize ();
+
+          /**
+           * Return LAB depth as a function of position (latitude and longitude). For the
+           * current class, this function returns value from the text file. We read in
```
Comment (gassmoeller) [inline include/aspect/initial_temperature/lithosphere_mask.h]: Please update the comment to say it either returns the value from the file, or a constant value depending on the input parameters.

File: tests/initial_condition_lithosphere_mask/screen-output [hunk 2]
```diff
@@ -1,5 +1,9 @@
+-----------------------------------------------------------------------------
+-----------------------------------------------------------------------------
 
-   Loading Ascii data lookup file ASPECT_DIR/data/initial-temperature/lithosphere_mask/LAB_CAM2016.txt.
+   Loading Ascii data lookup file /Users/scoulson/Documents/PhD/aspect_projects/aspect/data/initial-temperature/lithosphere-mask/LAB_CAM2016.txt.
```
Comment (gassmoeller) [inline tests/initial_condition_lithosphere_mask/screen-output]: Please revert this change by restoring the old line.

**Comment by SophieCoulson** [comment]
Done.

**Comment by gassmoeller** [comment]
Thank you!
